lib/router/router.js

```diff
@@ -276,7 +283,8 @@ export default class Router extends EventEmitter {
 }
 
 function getURL () {
-  return window.location.pathname + (window.location.search || '') + (window.location.hash || '')
+  const { href, origin } = window.location
+  return href.replace(new RegExp(`^${origin}`), '')
```

Comment:
Yeah. You mean using something like `indexOf` and `substring`?
If we simply use a simple string replace, it might replace the `origin` in somewhere else, may be in a query param.